src/storage/models.rs

```diff
@@ -60,6 +60,39 @@ impl Snippet {
 }
 
 #[derive(Debug)]
+pub enum Direction {
+    /// From newer to older snippets.
+    Backward,
+    /// From older to newer snippets.
+    #[allow(dead_code)]
```

Comment:
It's not plugged in to the API layer now (which does not exist yet :) ), so that's why it's dead. `Backward` is currently used in `default()`, which makes the compiler happy.